### ISSUE TITLE
Native launchers are not published for macOS on aarch64

### ISSUE THREAD

**Issue by julienrf** [sbt/sbt#7143]
In the release assets, the .zip file currently contains many `sbtn` native launchers, but the one for macOS and aarch64 is missing.

Would it be possible to build such native launchers, and to add them to the release assets?

We would also need to support that combination (`"$OSTYPE" == "darwin"` and `"$arch" == "aarch64"`) here:

https://github.com/sbt/sbt/blob/3f1ff987905620718825b55d28cdcb65f094e7ce/sbt#L175-L194

**Comment by eed3si9n** [comment]
<s>it's a universal binary for macOS.</s>

**Comment by julienrf** [comment]
Ah, interesting, do you mean that the file `bin/sbtn-x86_64-apple-darwin` also works on aarch64?

**Comment by mzuehlke** [comment]
I just tried the one from version 1.8.2 installed via SDKMAN:

`sbtn` is executable (but I have Rosetta installed) and the following indicates it is a x86 binary:
```bash
❯ file .sdkman/candidates/sbt/1.8.2/bin/sbtn-x86_64-apple-darwin
.sdkman/candidates/sbt/1.8.2/bin/sbtn-x86_64-apple-darwin: Mach-O 64-bit executable x86_64
```
![Bildschirm­foto 2023-01-27 um 18 41 40](https://user-images.githubusercontent.com/204523/215157076-bcd9f713-f9aa-4115-b4e9-43cf7c777b2b.png)

**Comment by eed3si9n** [comment]
> Ah, interesting, do you mean that the file `bin/sbtn-x86_64-apple-darwin` also works on aarch64?

No, I got confused with sbt/ipcsocket (https://github.com/sbt/ipcsocket/blob/ee3439303e592846168a3b18f23358d3c6665949/build.sbt#L36-L46), which we do try to create a universal binary, but sbtn is currently built using GraalVM native-image, and it's Intel-only for macOS.

**Comment by julienrf** [comment]
Hey @mzuehlke, @okennedy, and @fsvehla.
I’ve tried to set up a workflow to build binaries of sbtn for macOS on the ARM64 architecture in sbt/sbtn-dist#4. Could you please download and try the produced assets? You can download it from here: https://cirrus-ci.com/task/5665273223577600. (It seems you will have to approve its execution because it is not signed)

**Comment by mzuehlke** [comment]
I downloaded the artifact and execute it.
And I can confirm that this is an ARM64 executable 👍

**Comment by julienrf** [comment]
This should be fixed by sbt/sbtn-dist#4, which will be available in version 1.9.0 (you can already try it in [1.9.0-M1](https://github.com/sbt/sbtn-dist/releases/tag/v1.9.0-M1))
